### DISCUSSION TITLE
Java 19 support

### DISCUSSION THREAD

**Discussion by josh-hemphill** [redhat-developer/vscode-java#2684] [General]
Not sure where to look for ECJ support of Java 19, but I saw eclipse JDT language server added it, so I'm curious if there's any info on if when it'll be supported here. (Ultimately I'm trying to get completions to work for the Project Panama preview APIs)

**Comment by fbricon** [comment]
It's been available in the pre-release channel of vscode for the [past couple of weeks](https://twitter.com/fbricon/status/1567167429977014275). 
<img width="1588" alt="Screenshot 2022-09-20 at 19 29 30" src="https://user-images.githubusercontent.com/148698/191325163-e2bc455c-7695-442e-a216-3d1635964f56.png">
